doc/guide/running-software.md

```diff
@@ -1,3 +1,7 @@
+<!--
+Copyright lowRISC Contributors.
+SPDX-License-Identifier: Apache-2.0
+-->
 # Running Sonata Software
```

Comment:
Captialisation of 'sonata software' is inconsistent within body text in this repository. It is either 'Sonata software' or 'sonata software', never 'Sonata Software'. You should probably pick one and change titles to match as part of your sentence case titles campaign.

Reply:
Thanks, I chose Sonata software.